### PR TITLE
Khp3 6575 add ability to edit the relationship details once a relationship form has been saved

### PR DESCRIPTION
**Pull request by Omoshlawi**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots

https://github.com/user-attachments/assets/c45a5459-8a13-479e-823f-6d89d9a41d86

## Related Issue

https://thepalladiumgroup.atlassian.net/browse/KHP3-6575

## Other

*None.*

### CODE REVIEW

**Comment by ojwanganto** [comment]
Thanks, @Omoshlawi. We can add the patient/person name at the top of the form.

File: packages/esm-patient-clinical-view-app/src/relationships/forms/form.scss [hunk 1]
```diff
@@ -0,0 +1,72 @@
+@use '@carbon/styles/scss/spacing';
+@use '@carbon/styles/scss/type';
+@use '@carbon/layout';
+@use '@carbon/colors';
+@import '~@openmrs/esm-styleguide/src/vars';
```
Comment (donaldkibet) [inline packages/esm-patient-clinical-view-app/src/relationships/forms/form.scss:5]: We should start depricating the use of this as most of our styles are already provided by carbon

File: packages/esm-patient-clinical-view-app/src/relationships/forms/relationships-update-form.workspace.tsx [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+import { relationshipUpdateFormSchema, updateRelationship } from '../relationship.resources';
+import styles from './form.scss';
+
+interface RelationshipUpdateFormProps extends DefaultWorkspaceProps {
+  relationShipUuid: string;
+}
+
+type RelationshipUpdateFormType = z.infer<typeof relationshipUpdateFormSchema>;
+
+const RelationshipUpdateForm: React.FC<RelationshipUpdateFormProps> = ({ closeWorkspace, relationShipUuid }) => {
```
Comment (donaldkibet) [inline packages/esm-patient-clinical-view-app/src/relationships/forms/relationships-update-form.workspace.tsx:31]:
```suggestion
const EditRelationShipForm: React.FC<RelationshipUpdateFormProps> = ({ closeWorkspace, relationShipUuid }) => {
```

File: packages/esm-patient-clinical-view-app/src/hooks/useRelationship.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { FetchResponse, openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
+import useSWR from 'swr';
+import { Relationship } from '../types';
+
+const useRelationship = (relationshipUuid: string) => {
```
Comment (donaldkibet) [inline packages/esm-patient-clinical-view-app/src/hooks/useRelationship.ts:5]: could we consolidate these hooks under one resource file e.g 

```ts
relationships.resource.tsx
```

Where we can group all API related hooks and functionality relating to relationships

**Review by donaldkibet — APPROVED** [review]
Thanks @Omoshlawi, left some suggestions

**Comment by donaldkibet** [comment]
@Omoshlawi address the feedback in follow up PR. Thanks merging this in.

**Comment by Omoshlawi** [comment]
@ojwanganto , something like this??

image

**Comment by ojwanganto** [comment]
Exactly. Thank you

**Pull request merged**